shell/update-script-counts.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Pre-requisites for running this script:
+#
+# - bun (https://bun.sh)
+# - foundry (https://getfoundry.sh)
+
+# Strict mode
+set -euo pipefail
+
+# Path to the Solidity file
+SOL_FILE="script/Base.s.sol"
+
+# Compile the contracts with optimized profile
+bun run build:optimized
+
+# Generalized function to update counts in the solidity file
+update_counts() {
+    local test_name=$1
+    local map_name=$2
+    echo "Running forge test for estimating $test_name..."
+    local output=$(FOUNDRY_PROFILE=benchmark forge t --mt "test_Estimate${test_name}" -vv)
+    echo "Parsing output for $test_name..."
+
+    # Parse the output to extract counts and chain IDs
```

Comment:
Would you like to make the output look a bit more friendly?

<img width="592" alt="Screenshot 2024-05-10 at 00 09 32" src="https://github.com/sablier-labs/v2-core/assets/6676622/6bd905cb-006f-47bd-a6b9-fec5e6db35c6">

Reply:
Any suggestions? 
we can even completely remove the displayed message - as we do in Precompiles

Reply:
I think display message is useful. Let me give it a shot.

Reply:
Finally managed to make it look better. The command time (~5 mins is really a PITA for debugging).
<img width="375" alt="Screenshot 2024-05-11 at 17 39 14" src="https://github.com/sablier-labs/v2-core/assets/6676622/d53b9d31-e8db-4d77-ab70-fb5a9d49b999">